prometheus_api_client/utils.py

```diff
@@ -0,0 +1,21 @@
+"""
+Some helpful functions used in the API
+"""
+import dateparser
+
+
+def parse_datetime(date_string: str, settings: dict = None):
+    """
+    A wrapper for dateparser.parse, but the default settings are set
+    to {"DATE_ORDER": "YMD"}
+    """
+
+    settings = settings or {"DATE_ORDER": "YMD"}
```

Comment:
why not put the default in the function definition?
`def parse_datetime(date_string: str, settings: dict = {"DATE_ORDER": "YMD"}):`
?

Reply:
pylint screamed at me for it not being a safe thing to do.

Reply:
http://pylint-messages.wikidot.com/messages:w0102

Reply:
ok, 🤷‍♂